packages/common/rollup-configs.js

```diff
@@ -29,20 +31,21 @@ const getBanner = (pkg) => `/*
 /**
  * @returns {import('rollup').RollupOptions}
  */
-export const tsLibConfig = (pkg) => ({
-  input: 'src/index.ts',
+export const tsLibConfig = (pkg, inputFile) => ({
+  input: inputFile,
   output: {
     file: 'dist/index.js',
-    format: 'esm',
     banner: getBanner(pkg),
   },
   plugins: [
     nodeResolve(),
     typescript({
       tsconfig: './tsconfig.json',
+      include: ['src/**/*', '../common/src/**/*'],
     }),
     analyzer({
       summaryOnly: true,
+      root: rootDir,
```

Comment:
This makes the `rollup-plugin-analyzer` output module paths relative to monorepo root directory, since e.g. `packages/lib-utils` may include code from `packages/common`.